Research community open call blog post

mattosborn opened the pull request:
See #461

- [ ] Add YT embed link once the vid has been uploaded
- [ ] Add YT thumbnail/splash as the post image
- [ ] Add video transcription for the presentation part of the call

@ZybN or @nathenf:
1. Is it fine to use the "Life Itself" profile as the author for these posts?
2. Is the `commmunity-call` category appropriate? Or should these posts have a new category e.g. `research-community`? Tags if not?
3. Any feedback or changes to title, body, etc

rufuspollock commented on content/blog/2023/04/26/research-community-open-call.md:
```diff
@@ -0,0 +1,27 @@
+
+### Video
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
In theory you should just be able to post a youtube video link and it "just works" ... and this is fine too.

mattosborn replied:
FYI this feature requires this specific YouTube URL format or it will either not detect it or incorrectly parse it:

https://www.youtube.com/watch?v=VKKUBK4J6Ho

rufuspollock commented on content/blog/2023/04/26/research-community-open-call.md:
```diff
@@ -0,0 +1,27 @@
+  - lifexitself
+---
+
+This is the first in a series of monthly open meetings hosted by the Life Itself research community. These purpose of these calls is to provide an opportunity for newcomers to learn about the community and engage in discussions, and for community members to share updates about ongoing work and coordinate across interest groups.
```
Very nice!

rufuspollock replied:
I would probably link Life Itself research community to `/research` page (using obsidian syntax). Minor point.

rufuspollock commented on content/blog/2023/04/26/research-community-open-call.md:
```diff
@@ -0,0 +1,27 @@
+
+This is the first in a series of monthly open meetings hosted by the Life Itself research community. These purpose of these calls is to provide an opportunity for newcomers to learn about the community and engage in discussions, and for community members to share updates about ongoing work and coordinate across interest groups.
+
+In this opening call, Rufus proposes an initial framework for a research agenda aligned with Life Itself’s wider purpose of paradigmatic transition. This approach is informed by the [[primacy-of-being|primacy of being]] thesis and is primarily focused on the topics of inner development and cultural evolution, with a secondary focus on systems transformation and the politics of paradigmatic change ...
```
liked that you linked to this stuff.

rufuspollock commented on content/blog/2023/04/26/research-community-open-call.md:
```diff
@@ -0,0 +1,27 @@
+
+> You can find more detailed information about the research community—including our approach, focus topics and ongoing projects—on our [[research|overview page]].
+
+### How to participate
```
Formatting: normally spacing around headings, paragraphs etc - see https://www.datopian.com/playbook/style-guide/markdown if you want to get picky 😉

rufuspollock reviewed (approved):
Looks great 👍 👏

rufuspollock commented:
> Is it fine to use the "Life Itself" profile as the author for these posts?

Yes, it's fine though you could also be the author if you would like a credit 😄

> Is the commmunity-call category appropriate? Or should these posts have a new category e.g. research-community? Tags if not?

We aren't systematically tagging stuff yet - see #223 for ongoing discussion

> Any feedback or changes to title, body, etc

See approved PR

mattosborn commented:
- Updated based on your review comments.
- Updated the post creation date.
- Added your notes to the bottom of the post
- Added the (currently unlisted) YT video using the magic YT link detection rather than iframe embed format.

@rufuspollock This should be ready to publish now if you want to do another quick once over.

@ZybN YT video is currently unlisted [here](https://www.youtube.com/watch?v=VKKUBK4J6Ho). Could you do a quick review (title, description, etc), and then I'll make the YT vid public and merge this PR.

rufuspollock commented:
@mattosborn let's seek forgiveness rather than permission and just make the video live and we merge and get this live!

😄😄

rufuspollock commented:
@mattosborn i think if we do have a decent-ish transcript we can add that (later! let's ship this as is for now!)

rufuspollock commented:
🎉 live at https://lifeitself.org/blog/research-community-open-call-1-april-2023
